spec/rails_helper.rb

```diff
@@ -1,5 +1,9 @@
 require 'simplecov'
-SimpleCov.start 'rails'
+SimpleCov.start 'rails' do
+  add_filter %r{^/app\/bindings/}
+  add_filter { |src| src.filename =~ /swagger/ }
+  add_filter { |src| src.filename =~ /scout/ }
+end
```

Comment:
are we reporting coverage? can we?

Reply:
https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/711